src/components/ChooseCard.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import './ChooseCard.css';
+
+class ChooseCard extends Component {
```

Comment:
Choose is a verb and a component should have a declarative name. Thus, `CardSelection` or `CardChoice` would be better.